onnx/examples/make_model.ipynb

```diff
@@ -71,7 +71,7 @@
       "  }\n",
       "}\n",
       "opset_import {\n",
-      "  version: 6\n",
+      "  version: 7\n",
```

Comment:
Why do we only change this file?

Reply:
I was wondering why only this notebook needs changes. I think I know why now. Because, in other cases, we only load the existing model, not making. Only in this case, new model is created. :-)